build.default.props

```diff
@@ -178,7 +178,6 @@ behat.wd_host.url = http://selenium:4444/wd/hub
 
 # Binaries.
 # ---------
-project.bin.composer = composer.phar
```

Comment:
@jonhy81 could you tell me why this was done? My local development system relies on being able to change the composer binary location...

Reply:
`project.bin.composer` back to toolkit.